acceptance-tests/src/main/java/io/blueocean/ath/ATHJUnitRunner.java

```diff
@@ -122,8 +123,8 @@ private void writeScreenShotCause(Throwable t, Object test, FrameworkMethod meth
 
                 byte[] screenshot =  Base64.getMimeDecoder().decode(se.getBase64EncodedScreenshot());
 
-                Files.createParentDirs(file);
-                Files.write(screenshot, file);
+                Files.createDirectories(file.toPath());
```

Comment:
Thanks s looks wrong to me, like it will creat a directory for the screenshot so the subsequent writing will fail as the file is a directory

Reply:
oh good catch!!!

Reply:
fixed